financials/reports/funding_transparency_2020Q4.md

```diff
@@ -0,0 +1,69 @@
+# 2020Q4 Funding Transparency Report
+
+Last updated: Mar 14 2021
+
+## Introduction
+This document provides a high level overview of Grin's financials for Q4 of 2020 (Oct 01 - Dec 31). The purpose is to keep the community up to date on funding activity in transparent and digestible way. The data is derived from what is reported in the financial logs under [/grin-pm/financials](https://github.com/mimblewimble/grin-pm/tree/master/financials).
+
+### Notes
+
+1. Data was up to date at the time of writing, but it's still possible that there have been changes and updates in the logs since.
+2. USD conversion rates are derived from the closing price of the asset on the day in question according to Coingecko.com unless otherwise stated. 
+
+## Status before the period
+
+### Disposable funds
+
+At the end of Sep 30 2020, Grin held disposable funds to the equivalent of $1,127,130.25 in the following assets:
+
+Currency | Amount | USD x-rate Sep 30 2020 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+GRIN | 3770.71555489 | $0.306694 | $1,156.46 | https://donations.grin-tech.org
```

Comment:
https://donations.grin-tech.org is expired and currently used as a spam website.

Reply:
Good point. This should probably be addressed across all the old reports, and could then be raised in a distinct pull request. I've filed an issue to track this: https://github.com/mimblewimble/grin-pm/issues/406